fdbserver/LogRouter.actor.cpp

```diff
@@ -89,6 +90,7 @@ struct LogRouterData {
 	double maxWaitForVersionTime = 0;
 	double getMoreTime = 0;
 	double maxGetMoreTime = 0;
+	uint64_t generation = -1;
```

Comment:
Any reason to declare it uint64_t? -1 makes sense for in64_t. int64_t will be compatible with the return type of getValue() too?

Reply:
Ah good catch. Got too many interruptions when writing this LOL

Reply:
```suggestion
	int64_t generation = -1;
```